keps/sig-auth/2799-token-controller-deprecation/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: Token Controller Deprecation
+kep-number: 2799
+authors:
+  - "@zshihang"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+status: implementable
+creation-date: 2021-06-25
+reviewers:
+  - "@liggitt"
+approvers:
+  - "@liggitt"
+prr-approvers:
+  - TBD
+stage: alpha
+latest-milestone: "v1.23"
+milestone:
+  alpha: "v1.23"
+feature-gates:
+  - name: TokenControllerOptOut
```

Comment:
I'm concerned that when this feature is GA, its name implies that you can always opt out of using the token controller (and, by further implication, that if you do not opt out, the token controller remains present and active, because the stable feature is part of their cluster).

Because that's not quite right, I wonder if we can tweak the feature gate name to avoid misleading people.

I don't mind what the new name is so long as the GA meaning makes sense even long after graduation. My suggestion: `ServiceAccountTokenReconciliationControl`.

Reply:
since there is [new service account token](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume), ServiceAccountTokenReconciliationControl would be misleading sometimes. How about `LegacyServiceAccountTokenReconciliationControl`?